modules/administration/pages/backup-restore.adoc

```diff
@@ -1,5 +1,5 @@
 [[backup-restore]]
-= Backup and Restore
+= Backing Up and Restoring {productname}
```

Comment:
In general, we currently want to avoid -ing forms with section titles.  I'm not sure whether `{macro}` at the beginning of a title is possible.
```suggestion
= {productname} Backup and Restore
```
Feel free to ignore this proposal.  Maybe, it is too dogmatic...

Reply:
I'll not integrate this change, as naming in this case makes sense (sometimes we cannot avoid -ING, see Monitoring) at this level in the navigation.